oneflow/core/auto_parallel/sbp_constructor.h

```diff
@@ -34,25 +35,33 @@ class SbpConstructor final {
   SbpConstructor(const OpGraph& op_graph, Job* job)
       : cost_ratio_(job->job_conf().auto_parallel_computation_cost_ratio()),
         wait_time_(job->job_conf().auto_parallel_wait_time()),
-        transfer_cost_(job->job_conf().auto_parallel_transfer_cost()) {
+        transfer_cost_(job->job_conf().auto_parallel_transfer_cost()),
+        use_sbp_collector_(!Global<ResourceDesc, ForSession>::Get()
+                                ->resource()
+                                .disable_group_boxing_by_dst_parallel()) {
```

Comment:
这里导出一个 python api，然后弄成：
```suggestion
        use_sbp_collector_(!Global<ResourceDesc, ForSession>::Get()
                                ->resource()
                                .disable_group_boxing_by_dst_parallel() && job->job_conf().enable_auto_parallel_sbp_collector()) {
```